src/Yaapii.Atoms/Scalar/Or.cs

```diff
@@ -65,6 +65,13 @@ public Or(IFunc<In, Boolean> func, params In[] src) : this(func, new EnumerableO
             )
         { }
 
+        /// <summary> True if any functions return true with given input value </summary>
+        /// <param name="value"> Input value wich will executed by all given functions </param>
+        /// <param name="functions"> Functions wich will executed with given input value </param>
+        public Or(In value, params Func<In, bool>[] functions)
+            : this(tValue => new Or(new Mapped<Func<In, bool>, bool>(tFunc => tFunc.Invoke(tValue), functions)).Value(), value)
```

Comment:
please remove the t prefix and fix indentation (this has to be on the right side)

Reply:
Done